src/chain_sync/tipset_syncer.rs

```diff
@@ -1013,12 +1014,20 @@ fn sync_tipset<DB: Blockstore + Clone + Sync + Send + 'static, C: Consensus>(
     })
 }
 
-async fn fetch_batch<DB: Blockstore + Clone + Send + Sync + 'static, C: Consensus>(
-    batch: &[Arc<Tipset>],
+async fn fetch_batch<DB: Blockstore, C: Consensus>(
```

Comment:
```suggestion
/// Ask peers for the [`Message`]s that these [`Tipset`]s should contain
async fn fetch_batch<DB: Blockstore, C: Consensus>(
```

Maybe this is where the preamble about batch sizes should go